nodejs-react-bootstrap/nodejs-react-bootstrap.spec

```diff
@@ -9,68 +9,66 @@ License: MIT
-Source9: http://registry.npmjs.org/uncontrollable/-/uncontrollable-4.1.0.tgz
-Source10: http://registry.npmjs.org/core-js/-/core-js-2.5.1.tgz
-Source11: http://registry.npmjs.org/warning/-/warning-3.0.0.tgz
-Source12: http://registry.npmjs.org/dom-helpers/-/dom-helpers-3.2.1.tgz
-Source13: http://registry.npmjs.org/prop-types/-/prop-types-15.6.0.tgz
-Source14: http://registry.npmjs.org/js-tokens/-/js-tokens-3.0.2.tgz
-Source15: http://registry.npmjs.org/fbjs/-/fbjs-0.8.16.tgz
-Source16: http://registry.npmjs.org/object-assign/-/object-assign-4.1.1.tgz
-Source17: http://registry.npmjs.org/promise/-/promise-7.3.1.tgz
+Source2: http://registry.npmjs.org/prop-types-extra/-/prop-types-extra-1.0.1.tgz
+Source3: http://registry.npmjs.org/dom-helpers/-/dom-helpers-3.3.1.tgz
+Source4: http://registry.npmjs.org/classnames/-/classnames-2.2.5.tgz
+Source5: http://registry.npmjs.org/keycode/-/keycode-2.1.9.tgz
+Source6: http://registry.npmjs.org/prop-types/-/prop-types-15.6.0.tgz
+Source7: http://registry.npmjs.org/uncontrollable/-/uncontrollable-4.1.0.tgz
+Source8: http://registry.npmjs.org/babel-runtime/-/babel-runtime-6.26.0.tgz
+Source9: http://registry.npmjs.org/object-assign/-/object-assign-4.1.1.tgz
+Source10: http://registry.npmjs.org/react-overlays/-/react-overlays-0.7.4.tgz
+Source11: http://registry.npmjs.org/regenerator-runtime/-/regenerator-runtime-0.11.1.tgz
+Source12: http://registry.npmjs.org/core-js/-/core-js-2.5.3.tgz
+Source13: http://registry.npmjs.org/warning/-/warning-3.0.0.tgz
+Source14: http://registry.npmjs.org/loose-envify/-/loose-envify-1.3.1.tgz
+Source15: http://registry.npmjs.org/js-tokens/-/js-tokens-3.0.2.tgz
+Source16: http://registry.npmjs.org/fbjs/-/fbjs-0.8.16.tgz
+Source17: http://registry.npmjs.org/setimmediate/-/setimmediate-1.0.5.tgz
 Source18: http://registry.npmjs.org/isomorphic-fetch/-/isomorphic-fetch-2.2.1.tgz
-Source19: http://registry.npmjs.org/ua-parser-js/-/ua-parser-js-0.7.17.tgz
+Source19: http://registry.npmjs.org/promise/-/promise-7.3.1.tgz
 Source20: http://registry.npmjs.org/core-js/-/core-js-1.2.7.tgz
 Source21: http://registry.npmjs.org/asap/-/asap-2.0.6.tgz
-Source22: http://registry.npmjs.org/node-fetch/-/node-fetch-1.7.3.tgz
-Source23: http://registry.npmjs.org/whatwg-fetch/-/whatwg-fetch-2.0.3.tgz
+Source22: http://registry.npmjs.org/whatwg-fetch/-/whatwg-fetch-2.0.3.tgz
+Source23: http://registry.npmjs.org/node-fetch/-/node-fetch-1.7.3.tgz
 Source24: http://registry.npmjs.org/is-stream/-/is-stream-1.1.0.tgz
 Source25: http://registry.npmjs.org/encoding/-/encoding-0.1.12.tgz
 Source26: http://registry.npmjs.org/iconv-lite/-/iconv-lite-0.4.19.tgz
-Source27: http://registry.npmjs.org/setimmediate/-/setimmediate-1.0.5.tgz
+Source27: http://registry.npmjs.org/ua-parser-js/-/ua-parser-js-0.7.17.tgz
 Source28: react-bootstrap-0.31.5-registry.npmjs.org.tgz
 BuildRequires: nodejs-packaging
-BuildRequires: npm(react) >= 15.3.0
```

Comment:
These are here since they're peer dependencies. Not sure if that's relevant but I kept them in when I updated to be safe.

Reply:
I don't think that's needed anymore, but I can bring them back